cmd/operator-sdk/internal/genutil/genutil.go

```diff
@@ -45,8 +45,20 @@ func parseGroupVersions() (map[string][]string, error) {
 
 			gvs[g.Name()] = make([]string, 0)
 			for _, v := range versions {
-				if v.IsDir() && scaffold.ResourceVersionRegexp.MatchString(v.Name()) {
-					gvs[g.Name()] = append(gvs[g.Name()], v.Name())
+				if v.IsDir() {
```

Comment:
@LiliC This was added in #1525, which was merged recently.

Reply:
@estroz yes this was due to me removing a commit :) should be fine now.